env0/resource_cost_credentials.go

```diff
@@ -23,13 +23,6 @@ func resourceCostCredentials(providerName string) *schema.Resource {
 			ForceNew:    true,
 			Required:    true,
 		},
-		"external_id": {
```

Comment:
Removed also from cost credentials...
I was guessing this also needs to be removed. I hope this was correct.
(Worst case, I'll add it back).